evennia/objects/objects.py

```diff
@@ -1472,13 +1477,14 @@ def at_access(self, result, accessing_obj, access_type, **kwargs):
 
     # hooks called when moving the object
 
-    def at_pre_move(self, destination, **kwargs):
+    def at_pre_move(self, destination, move_type="move", **kwargs):
         """
         Called just before starting to move this object to
         destination.
 
         Args:
             destination (Object): The object we are moving to
+            move_type (str): The type of move. "give", "traverse", etc.
```

Comment:
This makes it sounds like there are only a few allowed types ... clarify that this is an _arbitrary_ name chosen by the event that triggers the move.